e2etest/e2etest_suite_test.go

```diff
@@ -98,8 +103,20 @@ var _ = ginkgo.BeforeSuite(func() {
 		testNameSpace = ns
 	}
 
-	if dir := os.Getenv("FRR_CONFIG_DIR"); len(dir) != 0 {
-		frrTestConfigDir = dir
+	if net := os.Getenv("FRR_CONTAINER_NETWORK"); len(net) != 0 {
+		containerNetwork = net
+		if containerNetwork == "host" {
+			if ip := os.Getenv("PROVISIONING_HOST_EXTERNAL_IPv4"); len(ip) != 0 {
```

Comment:
just put it all uppercase

Reply:
no need to be under containerNetwork == "host", no need to check the ip (at least here). I'd do the full validation here https://github.com/metallb/metallb/pull/1008/files#diff-a115369e9006a64f1497f61daada01a4136fa15a1feb79f86f11a73a0cdc5d36R68 and fail if some bits are missing